controllers/admin/AdminTabsController.php

```diff
@@ -33,15 +40,18 @@ class AdminTabsControllerCore extends AdminController
 
     public function __construct()
     {
+        // construct parent just to have the translator :D
+        parent::__construct();
+
+        die($this->trans('This functionality has been disabled.', [], 'Admin.Notifications.Error'));
+
```

Comment:
if we put a `die()` here, isn't all the following code useless?

Reply:
Agree with you. But in the issue, it was asked to keep the code so we can activate it on demand

Reply:
Maybe add a comment to explain it?

Reply:
👍🏾

Reply:
oh ok @sowbiba

Reply:
well one could argue that version control is also made for this :D. 
But it's no big deal